python/sglang/test/runners.py

```diff
@@ -147,7 +147,9 @@ def start_model_process(self, in_queue, out_queue, model_path, torch_dtype):
             ).cuda()
         else:
             raise Exception(f"Unrecognized model type {self.model_type}")
-        self.tokenizer = get_tokenizer(model_path, torch_dtype=torch.dtype)
+        self.tokenizer = get_tokenizer(
+            model_path, torch_dtype=torch.dtype, trust_remote_code=True
+        )
```

Comment:
Is this change necessary? Many other tests use this code. It would be better to keep the original version.

Reply:
(see above)